### DISCUSSION TITLE
Issue With plex_search

### DISCUSSION THREAD

**Discussion by themoops2021** [meisnate12/Plex-Meta-Manager#143] [Q&A]
Trying to build some genre and decade collections with plex_search, but I'm getting same error for each genre. 

The search by year or decade collections also gives the same error. Log and config linked below. Thanks.

[Log Error](https://pastebin.com/NKzQQbq9)

[Movies.yml](https://pastebin.com/G6hxXrBe)

**Comment by meisnate12** [comment]
this should be fixed with 1.6.2

**Reply by themoops2021** [reply]
Yeah, that fixed it for the genre search. Thanks. 

Still getting an error for the year and decade search

![image](https://user-images.githubusercontent.com/81425781/112719860-1d050f00-8ef3-11eb-92a8-ca2ed48f93dc.png)

Looks like the docker container also stops after each run, not sure if this is an issue with the script or Unraid though

**Reply by meisnate12** [reply]
fix is in develop

**Reply by themoops2021** [reply]
yeah working okay now.
